project-management/zenhub.md

```diff
@@ -0,0 +1,83 @@
+# Working with ZenHub
+
+[ZenHub](https://zenhub.com/) is "agile project management within Github".
+We use it for all squads in the design team.
+
+Use [app.zenhub.com](https://app.zenhub.com), it's much nicer :).
+
+Current squad boards:
+
+- [Snappy squad tasks](https://app.zenhub.com/workspace/o/ubuntudesign/snappy-squad-tasks/boards)
+- [Web squad](https://app.zenhub.com/workspace/o/ubuntudesign/web-squad/boards)
+- [MAAS design squad](https://app.zenhub.com/workspace/o/ubuntudesign/maas-design-squad/boards)
+- [Juju squad](https://app.zenhub.com/workspace/o/ubuntudesign/juju-squad/boards)
+- [Vanilla squad](https://app.zenhub.com/workspace/o/ubuntudesign/vanilla-squad/boards)
+
+## 1. Iterations
```

Comment:
Why have you numbered your headings? I'm not sure this is useful - it makes it more cumbersome to add new sections.

Reply:
I had grand plans, alas I lost stream

Reply:
If you think numbered headings add something then we should write all our docs this way. Otherwise we shouldn't. If you want, start an issue about it? Or submit a PR to edit CONTRIBUTING.md?

Reply:
removed numbering, too much overhead